spec.emu

```diff
@@ -218,14 +252,12 @@ location: https://github.com/tc39/proposal-measure/
           1. If _v_ < 0, let _prefix_ be *"-"*, else let _prefix_ be "".
           1. If _v_ < 0, set _v_ to -_v_.
           1. Let _rounded_ be ApplyRoundingModeToPositive(_v_ × 10<sup>_numDigits_</sup>, _mode_).
-          1. Set _v_ to _rounded_ × 10<sup>-_numDigits_</sup>.
           1. Let _e_ be the smallest non-negative integer such that _v_ × 10<sup>-_numDigits_</sup> is an integer.
-          1. Let _s_ be the unique decimal string representation of _v_ without leading zeroes.
-          1. If _numDigits_ > _e_, then
-            1. If _v_ is an integer, return the string concatenation of _prefix, _s_, *"."*, and *"0"* repeated _numDigits_ times.
-            1. Otherwise, return the string concatenation of _prefix_, _s_ and *"0"* repeated _numDigits_ - _e_ times.
-          1. Otherwise:
-            1. Return the string concatenation of _prefix, _s_ and *"0"* repeated _e_ - _numDigits_ times.
+          1. Set _v_ to _rounded_ × 10<sup>-_e_</sup>.
+          1. If _v_ is 0, let _s_ be *"0"*, otherwise let _s_ be the unique decimal string representation of _v_ without leading zeroes.
```

Comment:
I think phrasing like this can be more precise, but don't have a concrete suggestion at the moment.